### DISCUSSION TITLE
Google calendar how to fetch events

### DISCUSSION THREAD

**Discussion by crwjames** [iEnglishClub-tw/springI18n#5] [Q&A]
想要知道fetch events 跟show在Calendar banner上的操作教學

**Comment by victorDeveloper0821** [comment]
## 前端改動：
目前有修正上一頁與下一頁的按鈕，分享一些調整的思路：
 - 發現 css 樣式定義在 bootstrap 要 override 原本 `.carousel-control-next,.carousel-control-prev` 兩 class 樣式
 - 於 main.css 新增樣式將原本的定義 overrides 的 class 名稱（這邊是 `arrow-button`） 
 - 於 calendar.js 的 drawUI 中新增按鈕客製化的樣式 `arrow-button`，這樣就可以將上一頁與下一頁的按鈕樣式調整完成

可以詳見 [calendar_page_fix](https://github.com/iEnglishClub-tw/officialWebsite/tree/calendar_page_fix) 分支中的改動，包括 JS 註解新增、新增 `main.css` 的 `arrow-button` 複寫 bootstrap 的部分樣式

P.S. 這次前端有額外修正 calendarItem 內容被上下頁按鈕遮蔽問題，因此新增 function 控制 calendarItem 的左右邊縮排寬度

## 後端 Java 改動方式：
如果是需要修改查詢的邏輯建議在 java 修正

P.S. 這個是 google calendar 的 API doc，其中有一些格式與參數的設定；可以參考此[連結](https://developers.google.com/calendar/api/v3/reference/events/list)
